docs/content/guides/internationalization/language.md

```diff
@@ -136,12 +165,12 @@ ReactDOM.render(<ExampleComponent />, document.getElementById('example1'));
 
 ::: only-for react
 ::: tip
-Note that the `language` property is bound to the component separately using `language={this.language}"`, but it could be included in the `settings` property just as well.
+Note that the `language` property is bound to the component separately using `language={language}"`, but it could be included in the `settings` property just as well.
```

Comment:
```suggestion
Note that the `language` property is bound to the component separately using `language={language}`, but it could be included in the `settings` property just as well.
```